packages/neuron-ui/src/components/WalletSetting/index.tsx

```diff
@@ -98,62 +104,65 @@ const WalletSetting = ({
 
   useOnWindowResize(toggleBottomBorder)
 
-  const [createWallet, ...importWallet] = buttons
-
   return (
-    <div className={styles.container}>
-      <ChoiceGroup
-        className={styles.wallets}
+    <div>
+      <RadioGroup
+        defaultValue={currentID}
+        onChange={handleChange}
+        itemClassName={styles.radioItem}
         options={wallets.map(wallet => ({
-          key: wallet.id,
-          text: wallet.name,
-          checked: wallet.id === currentID,
-          onRenderLabel: props => {
-            return (
-              <span
-                role="presentation"
-                className={`ms-ChoiceFieldLabel ${styles.choiceLabel}`}
-                data-id={wallet.id}
-                data-action="select"
-                onClick={onHandleWallet}
-              >
-                <span className={styles.walletName}>{props?.text}</span>
-
-                <button type="button" data-action="edit" aria-label={t('common.edit')} title={t('common.edit')}>
-                  <EditWallet />
-                </button>
-                <button type="button" data-action="delete" aria-label={t('common-delete')} title={t('common.delete')}>
+          value: wallet.id,
+          label: <span className={styles.walletName}>{wallet.name}</span>,
+          suffix: (
+            <div className={styles.suffix}>
+              <button type="button" aria-label={t('common.edit')} onClick={() => handleEdit(wallet.id)}>
+                <EditWallet />
+              </button>
+              <button type="button" aria-label={t('common.backup')} onClick={() => onHandleWallet('backup', wallet.id)}>
```

Comment:
This function is better to get the `id` from `e`, then we will not create an arrow function every time.